### PR TITLE
fix path length calculation

### PR DESCRIPTION
**Pull request by xmyqsh**
s = v0 * t + 1 / 2.0 * a * t^2

### CODE REVIEW

**Comment by xmyqsh** [comment]
`
//cyber/timer:timer_test                                                 FAILED in 0.4s
  /home/travis/.cache/bazel/_bazel_travis/540135163923dd7d5820f3ee4b306b32/execroot/apollo/bazel-out/local-dbg/testlogs/cyber/timer/timer_test/test.log
`

What's wrong with it? 
The timer_test FAILED

**Comment by Capri2014** [comment]
@xmyqsh please fix the lint issue

File: modules/planning/navi/decider/navi_path_decider.cc [hunk 1]
```diff
@@ -240,7 +240,8 @@ bool NaviPathDecider::GetBasicPathData(
   double min_path_len = config_.min_path_length();
   // get min path plan length s = v0 * t + 1 / 2.0 * a * t^2
   double path_len = start_plan_v_ * config_.min_look_forward_time() +
-                    start_plan_a_ * pow(config_.min_look_forward_time(), 2) / 2.0;
+                    start_plan_a_ *
+                    pow(config_.min_look_forward_time(), 2) / 2.0;
```
Comment (xmyqsh) [inline modules/planning/navi/decider/navi_path_decider.cc]: @Capri2014 Lint issue has been fixed here.

Comment (xmyqsh) [reply modules/planning/navi/decider/navi_path_decider.cc]: @Capri2014 The Build Failed issue below is caused by `apollo.sh test`, not lint.
[5921.5](https://travis-ci.com/ApolloAuto/apollo/jobs/201384753#L1790-L1791) "JOB=citest_extended""JOB=citest_extended" | failed

**Pull request merged**